### DISCUSSION TITLE
Welcome to DataStorm DAQ Discussions!

### DISCUSSION THREAD

**Discussion by arrowperfecto** [ArrowElectronics/data-storm-daq#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * While you may use this discussion to ask questions of the community or owners of this solution, you can always email the support team directly at gc.na.fpga_platform_support@arrow.com.  This email is monitored and an Arrow resource will respond.  You can also submit technical support problems via the [Issues](https://github.com/ArrowElectronics/data-storm-daq/issues) feature.

**Comment by erfaren** [comment]
Is there a recommended part number for the FPGA fan and part number for J16 on the DataStorm DAQ FPGA?
